Assets/Source/Application/States/Play/PassthroughVideoCapture.cs

```
@@ -23,6 +24,12 @@ public IAsyncToken<Void> Setup(VideoAudioSource audioSource)
             return new AsyncToken<Void>(new NotSupportedException());
         }
 
+        /// <inheritdoc />
+        public IAsyncToken<Void> Setup(int width, int height, int framerate)
+        {
+            return new AsyncToken<Void>(new NotSupportedException());
```

In the rest of the code base we use "passthrough" to mean silently do nothing-- like a noop. If the implementation throws exceptions it would better be called `ExceptionVideoCapture`.